extjsdk/build.gradle

```diff
@@ -81,6 +81,9 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
     testCompile 'io.vantiq:vantiq-sdk:1.0.24'
+
+    // Used to create EvictingQueue for failed messages queue
+    implementation 'com.google.guava:guava:23.0'
```

Comment:
Tweak, but if we're moving here from `compile` to `implementation`, should we change the ones above as well?